### PR TITLE
Add GTP sodium salt (NaGTP)

### PR DESCRIPTION
**Pull request by apdavison**
Encountered as "NaGTP", a search for this term led to this [product page](https://www.sigmaaldrich.com/FR/fr/product/sigma/51120) which then led to this [PubChem page](https://pubchem.ncbi.nlm.nih.gov/compound/135818278)

### CODE REVIEW

File: instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/GTPDisodiumSalt",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "Guanosine 5'-triphosphate (GTP) sodium salt hydrate.",
```
Comment (UlrikeS91) [inline instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld:7]: Could we adjust this similarly to the [MgATP instance](https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/terminologies/molecularEntity/MagnesiumATP.jsonld)? 

Suggestion:
"A GTP (guanosine 5'-triphosphate) molecule bonded with two sodium ions (Na+) forming a salt hydrate."

File: instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "GTP, Disodium salt",
```
Comment (UlrikeS91) [inline instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld:11]: I think disodium should not be capitalized.

File: instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "preferredOntologyIdentifier": null,
```
Comment (UlrikeS91) [inline instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld:12]: The [MgATP instance](https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/terminologies/molecularEntity/MagnesiumATP.jsonld) states the pubchem references as preferred ID. Could you please add, the one from your description here?
 
Add: https://pubchem.ncbi.nlm.nih.gov/compound/135818278

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld (hunk 1 above)
Comment (apdavison) [reply instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld]: done

File: instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld (hunk 2 above)
Comment (apdavison) [reply instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld]: The capitalisation matches the PubChem entry, but I agree.

File: instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld (hunk 3 above)
Comment (apdavison) [reply instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld]: PubChem isn't an ontology, but I've followed your suggestion until we can find a better entry. I can't find the sodium salt of GTP in ChEBI.

Comment (UlrikeS91) [reply instances/latest/terminologies/molecularEntity/GTPDisodiumSalt.jsonld]: Yeah, I think that was the logic that @lzehl used for MgATP.

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**